### PR TITLE
Remove demo signer and add CSP to view.php

### LINKED ISSUE #14: Add CSP to View.php

### LINKED ISSUE #15: Remove demo_signer
The functionality has been moved to https://github.com/brave-experiments/sku-php

### PR DESCRIPTION
**Pull request by jumde**
Fix #14 
Fix #15

### CODE REVIEW

File: demo_cart/View.php [hunk 1]
```diff
@@ -4,6 +4,7 @@
     <meta charset="utf-8" />
     <meta http-equiv="x-ua-compatible" content="ie=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta http-equiv="Content-Security-Policy" content="script-src https://test-sku-store.bravesoftware.com">
```
Comment (diracdeltas) [inline demo_cart/View.php]: minor: if you do `self` instead of `https://test-sku-store.bravesoftware.com` then this works when the host changes

Comment (diracdeltas) [reply demo_cart/View.php]: `'self'` not `self` :)

Comment (jumde) [reply demo_cart/View.php]: Updated: 

https://github.com/brave-experiments/sku-js-php/pull/16/files#diff-bf158698768672c793cd0611ae1896a9R7

https://github.com/brave-experiments/sku-js-php/pull/16/files#diff-510a2c6d83825a13f81146c5008d6eb8R7

File: README.md [hunk 2]
```diff
@@ -44,9 +44,8 @@ The JS library adds an onload handler which parses the SKU tokens embedded in th
...
 3. Pass this [`SECRET`](https://github.com/brave-experiments/sku-js-php/blob/master/demo_cart/Controller.php#L17) to `generateSkuToken`.
 4. Generate sku tokens for each item. See sample code [here](https://github.com/brave-experiments/sku-js-php/blob/master/demo_cart/Controller.php#L25)
-5. Embed the `sku_tokens` in the web page. See sample code [here](https://github.com/brave-experiments/sku-js-php/blob/master/demo_cart/View.php#L59)
-6. Invoke `BraveSKU.init()` on the client side. See sample code [here](https://github.com/brave-experiments/sku-js-php/blob/master/demo_cart/View.php#L19). Init parses the SKU tokens to add a `Buy with BAT` button which on click invokes the Payment UI.
+5. Embed the `sku_tokens` in the web page. See sample code [here](https://github.com/brave-experiments/sku-js-php/blob/master/demo_cart/View.php#L56)
+6. Invoke `BraveSKU.init()` on the client side. See sample code [here](https://github.com/brave-experiments/sku-js-php/blob/master/demo_cart/js/init.js#L2). Init parses the SKU tokens to add a `Buy with BAT` button which on click invokes the Payment UI.
```
Comment (corymcdonald) [inline README.md]: It might be better if we "Copy permalink" so that the links in the README will never 404?

Comment (jumde) [reply README.md]: These links will work when the code is merged, I wanted the README.md to point to the latest code. Let me know if you have any concerns.

**Pull request merged**